corehq/apps/export/models/new.py

```diff
@@ -764,6 +858,19 @@ class FormExportInstance(ExportInstance):
     # Whether to include duplicates and other error'd forms in export
     include_errors = BooleanProperty(default=False)
 
+    def __init__(self, *args, **kwargs):
+        super(FormExportInstance, self).__init__(*args, **kwargs)
+        # Conditional default values for self.filters
+        created_default_filters = "filters" not in kwargs and (len(args) == 0 or "filters" not in args[0])
+        if created_default_filters:
+            self.filters.user_types = [0]
```

Comment:
this is kind of magical to me. why can't the default value of user_types by `[0]`. also a constant instead of 0 would be helpful